cds_ils/ldap/tasks.py

```diff
@@ -32,3 +32,20 @@ def synchronize_users_task():
         db.session.rollback()
         current_app.logger.exception(e)
         log.set_failed(e)
+
+
+@shared_task
+def anonymize_users_task():
```

Comment:
Question: I think that at the beginning, we added the `synchronize_users` method call to avoid a double fetch of users from LDAP.
Shall we do the same to be more efficient, instead of having a different celery task?

Reply:
I was thinking having separate task would be better in terms of separation of concerns. Especially if one of the tasks will be blocked (f.e. we cannot delete for some ldap related reasons), the import of new accounts would be blocked. What do you think ?

Reply:
I like the separation of concerns in the 2 tasks and I agree with @kprzerwa in the non-blocking of the 2 tasks, anonymize and import of users. I think that if fetching twice the ldap users is a problem then we need to find another solution to cache the fetched ldap users and used them across our tasks. But this is something we need to do only if it's causing trouble I think.

Reply:
Good point! 👏🏼